chat-commands.js

```diff
@@ -1990,7 +1990,8 @@ exports.commands = {
 		if (!this.can('declare', null, room)) return false;
 		if (!this.canTalk()) return;
 
-		this.add('|raw|<div class="broadcast-blue"><b>' + Chat.escapeHTML(target) + '</b></div>');
+		this.add(`|notify|The staff in ${room.id} made a declaration|${Chat.escapeHTML(target)}`);
```

Comment:
And iirc notification titles are usually are really short. So this might just be `${room.title} announcement!`